README.md

```diff
@@ -26,9 +29,9 @@ The token contract represents a standard fungible token contract with minting an
 |`owner` | Current owner of the contract initialized to `init_owner`. Certain critical actions can only be performed by the `owner`, e.g., changing who plays certain roles in the contract. |
 |`pauser` | Account that is allowed to (un)pause the contract. It is initialized to `init_owner`.  `pauser` can (un) pause the contract. There is only `pauser` for the contract. |
 |`masterMinter` | The master minter to manage the minters for the contract.  `masterMinter` can add or remove minters and configure the number of tokens that a minter is allowed to mint. There is only one `masterMinter` for the contract. |
-| `minter` | An account that is allowed to mint and burn new tokens. The contract defines several minters. Each `minter` has a quota for minting new tokens. |
-| `blacklister` | An account that can freeze, unfreeze & wipe the balance from any other account when required to do so by law enforcement. The presence of this function in the code is a mandatory regulatory requirement. StraitsX will never use this function on its own accord. There is only one `blacklister`. |
-|`approvedSpender`| A token holder can designate a certain address to send up to a certain number of tokens on its behalf. These addresses will be called `approvedSpender`.  |
+|`minter` | An account that is allowed to mint and burn new tokens. The contract defines several minters. Each `minter` has a quota for minting new tokens. |
```

Comment:
Add  more comments saying that the way to detect a minter is by looking at the minterAllowance since there is no longer a minter Boolean map

Reply:
```
|`minter`| An account that is allowed to mint and burn new tokens. The contract defines several minters in `minterAllowed` field. Call `minterAllowance` for adding a new `minter`. Each `minter` has a quota for minting new tokens. |
```

Reply:
```suggestion
|`minter`| An account that is allowed to mint and burn new tokens. The contract defines several minters in `minterAllowed` field. Call `minterAllowance` for adding a new `minter`. Each `minter` has a quota for minting new tokens. |
```